README.md

```diff
@@ -154,7 +154,7 @@ rss-xxx.tgz will be generated for deployment
 ### Support Spark dynamic allocation
 
 To support spark dynamic allocation with Firestorm, spark code should be updated.
-There are 2 patches for spark-2.4.6 and spark-3.1.2 in spark-patches folder for reference.
+There are 3 patches for spark-2.4.6, spark-3.1.2 and spark-3.2.1 in spark-patches folder for reference.
```

Comment:
spark (2.4.6/3.1.2/3.2.1)

Reply:
why do we need this style?

Reply:
The original style can be too long, When are there more minor versions

Reply:
OK